src/filters/default-filter.js

```diff
@@ -1,9 +1,9 @@
 import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 
-import TextField from '@material-ui/core/TextField'
+import { TextField } from '@eqworks/lumen-ui'
```

Comment:
since `lumen-table` doesn't explicitly handle tree-shaking, we should import individual component similarly as we imported from `@material-ui/core`